packages/extension-chakra-storefront/package.json

```diff
@@ -17,82 +17,76 @@
-    "react-intl": "^5.25.1",
-    "react-router-dom": "^5.3.4"
-  },
-  "peerDependenciesMeta": {
-    "@salesforce/pwa-kit-extension-sdk": {
-      "optional": true
-    },
-    "@salesforce/pwa-kit-react-sdk": {
-      "optional": true
-    }
-  },
-  "dependencies": {
-    "@chakra-ui/icons": "^2.0.19",
-    "@chakra-ui/react": "2.8.2",
-    "@chakra-ui/skip-nav": "^2.0.15",
-    "@chakra-ui/system": "^2.5.6",
     "@peculiar/webcrypto": "^1.4.2",
-    "@salesforce/commerce-sdk-react": "3.2.0-extensibility-preview.4",
-    "@salesforce/pwa-kit-runtime": "4.0.0-extensibility-preview.4",
+    "@salesforce/commerce-sdk-react": "^3.2.0-extensibility-preview.4",
+    "@salesforce/pwa-kit-dev": "^4.0.0-extensibility-preview.4",
+    "@salesforce/pwa-kit-extension-sdk": "^4.0.0-extensibility-preview.4",
+    "@salesforce/pwa-kit-react-sdk": "^4.0.0-extensibility-preview.4",
+    "@salesforce/pwa-kit-runtime": "^4.0.0-extensibility-preview.4",
     "@tanstack/react-query": "^4.28.0",
     "@tanstack/react-query-devtools": "^4.29.1",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^14.0.0",
+    "@testing-library/user-event": "14.5.2",
+    "@types/express": "^5.0.0",
+    "@types/jest": "29.5.14",
+    "@types/loadable__component": "^5.13.9",
+    "@types/mocha": "^10.0.9",
+    "@types/react": "~18.2.0",
+    "@types/react-dom": "~18.2.1",
+    "@types/react-helmet": "^6.1.11",
+    "@types/react-router-dom": "^5.3.3",
     "card-validator": "^8.1.1",
-    "cross-env": "^5.2.1",
     "cross-fetch": "^3.1.4",
+    "express": "^4.19.2",
     "focus-visible": "^5.2.0",
-    "framer-motion": "^10.12.9",
-    "full-icu": "^1.5.0",
     "helmet": "^4.6.0",
     "jest-fetch-mock": "^2.1.2",
-    "js-cookie": "^3.0.1",
     "jsonwebtoken": "^9.0.0",
-    "jwt-decode": "^4.0.0",
     "lodash": "^4.17.21",
     "msw": "^1.2.1",
     "nanoid": "^3.3.4",
     "prop-types": "^15.8.1",
     "query-string": "^7.1.3",
     "raf": "^3.4.1",
-    "randomstring": "^1.2.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-helmet": "^6.1.0",
     "react-hook-form": "^7.43.9",
     "react-intl": "^5.25.1",
     "react-router-dom": "^5.3.4"
   },
-  "devDependencies": {
+  "peerDependencies": {
+    "@chakra-ui/react": "^2.8.2",
+    "@chakra-ui/skip-nav": "^2.0.15",
     "@loadable/component": "^5.15.3",
-    "@salesforce/pwa-kit-dev": "4.0.0-extensibility-preview.4",
-    "@salesforce/pwa-kit-extension-sdk": "4.0.0-extensibility-preview.4",
-    "@salesforce/pwa-kit-react-sdk": "4.0.0-extensibility-preview.4",
+    "@peculiar/webcrypto": "^1.4.2",
+    "@salesforce/commerce-sdk-react": "^3.2.0-extensibility-preview.0",
+    "@salesforce/pwa-kit-extension-sdk": "^4.0.0-extensibility-preview.0",
+    "@salesforce/pwa-kit-react-sdk": "^4.0.0-extensibility-preview.0",
+    "@salesforce/pwa-kit-runtime": "^4.0.0-extensibility-preview.0",
+    "@tanstack/react-query": "^4.28.0",
     "@tanstack/react-query-devtools": "^4.29.1",
```

Comment:
Should we remove `@tanstack/react-query-devtools` from  peerDependencies and add it as a devDependencies since t’s not a runtime peer requirement?

Reply:
It is actually a runtime tool. You can see that it's always added in the `withCommerceSDKReact` hoc